src/pages/Download/Download.styles.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import React from 'react';
+import type { ComponentType } from 'react';
+
+
+export const Loading = ({ className, ...props }) => (
+    <p className={ `govuk-body govuk-!-margin-top-6 ${className}` } { ...props }/>
+);
```

Comment:
Check [this](https://github.com/publichealthengland/coronavirus-dashboard/blob/v3-development/src/components/Map/Map.styles.js#L270-L281) out for a more versatile / generic implementation of this.

Reply:
done